packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1062,13 +1104,21 @@ class RoundSliderThumbShape extends SliderComponentShape {
 ///    sliders in a widget subtree.
 class RoundSliderOverlayShape extends SliderComponentShape {
   /// Create a slider thumb overlay that draws a circle.
-  const RoundSliderOverlayShape();
+  const RoundSliderOverlayShape({this.overlayRadius});
+
+  /// The preferred radius of the round thumb shape when enabled.
+  ///
+  /// If it is not provided, then half of the track height is used.
+  final double overlayRadius;
+  double get _overlayRadius =>  overlayRadius ?? _defaultOverlayRadius;
```

Comment:
Same as above:

I think it would be better the get rid of `_overlayRadius` and instead have the constructor do:
```
const RoundSliderOverlayShape({this.overlayRadius = _defaultOverlayRadius});
```

Or, you could also just do 
```
const RoundSliderOverlayShape({this.overlayRadius = 16.0});
```
and add a comment above the constructor about the `16.0` coming from the Material spec.

Reply:
Done, and updated the thumb shape similarly